### PR TITLE
Picture tag

### PR DESCRIPTION
**Pull request by dutchenkoOleg**
### Summary

*Example with picture tag*

Create _a broken_ picture element structure.  

> IE browser don't support picture tag!  
> You need to set `data-iesrc` attribute (only for your picture tags) with source for IE browser

```html
<!-- For an element to be caught, add a block type that is different from the inline and some min-height for correct caught into view -->
<picture class="lozad" style="display: block; min-height: 1rem" data-iesrc="images/thumbs/04.jpg">
    <source srcset="images/thumbs/04.jpg" media="(min-width: 1280px)">
    <source srcset="images/thumbs/05.jpg" media="(min-width: 980px)">
    <source srcset="images/thumbs/06.jpg" media="(min-width: 320px)">
    <!-- NO img element -->
    <!-- instead of img element, there will be the last source with the minimum dimensions -->
    <!-- for disabled JS you can set <noscript><img src="images/thumbs/04.jpg"></noscript> -->
</picture>
``` 

When _lozad_ load this picture element, it will fix it.  
That's all ))

### CODE REVIEW

**Comment by dutchenkoOleg** [comment]
found that not work in IE ((  
added new commits, now it works

File: demo/picture.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE HTML>
```
Comment (ApoorvSaxena) [inline demo/picture.html]: @dutchenkoOleg what's the need to add a complete new demo page for `picture` tag use?

**Comment by iamdubx** [comment]
Is it implemented yet?

File: demo/picture.html (hunk 1 above)
Comment (dutchenkoOleg) [reply demo/picture.html]: I created new demo page for my own local testing.

In index demo page you have hardcoded style elements with inheritance from the id selector.  
Also you use custom `load` method for demonstrations of loading process, which overrided  
default load _logic_ with picture tag implementation

**Comment by dutchenkoOleg** [comment]
Fixed problems across forks

File: demo/picture.html (hunk 1 above)
Comment (ApoorvSaxena) [reply demo/picture.html]: please make use of the existing demo page, you can add a new CSS rule as per requirement. Also, you ca have multiple instances of Lozad on the same page.

Comment (dutchenkoOleg) [reply demo/picture.html]: Sorry for waiting. Just updated

**Comment by ApoorvSaxena** [comment]
thanks @dutchenkoOleg 👍

**Pull request merged**